.github/pull_request_template.md

```diff
@@ -1,45 +1,19 @@
-<!--
-Thank you for submitting a PR to the rust rpm implementation!
--->
-
-# PR Content Desc
```

Comment:
If the user creates a PR with only a single commit, their commit message will be put at the top of the file, and then it needs to be copied down below.

Reply:
Commit messages tend to be more brief than a content description, I am generally trying to optimize for very little version inquiry from the maintainer pov while still managing a decent experience for non-professionals/junior devs that need more guidance.

Reply:
To be clear, I'm not suggesting that they omit the description, the comments below still request that.  Just that the description header (this specific line) unnecessary and gets in the way a little bit.